# Historical genomics of complex traits in soybean and rice

## Project Summary

Crop improvement represents a long-running experiment in artificial selection on a complex trait, namely yield. How such selection relates to natural populations is unclear, but the analysis of domesticated populations could offer insights into the relative role of selection, drift, and recombination in all species facing major shifts in selective regimes. Because of the extreme autogamy exhibited by some crop species, many “immortalized” genotypes of elite varieties spanning the last century have been preserved and characterized either through dense genotyping or whole-genome resequencing.

Nearly all crops have undergone a population bottleneck and consequent loss of genetic diversity relative to their wild progenitors.  In soybean, genetic diversity across global germplasm is effectively half that of its wild progenitor.  This bottleneck can be a consequence of the fact that only a small family or, in the extreme case, a single individual, possess the desired features for agricultural production. Such scenarios establish genome-wide linkage disequilibrium (LD) between neutral alleles and those alleles underlying desirable traits, and this LD will substantially complicate the interpretation of loci identified by solely comparing pre- with post-bottleneck populations. Still, knowledge of these patterns is important for informing diversity enrichment strategies. Careful characterization of the post-founding population structure should allow us to more effectively differentiate reductions in diversity related to the bottleneck from those related to selection that occurred later, during modern improvement.

In this project, we use emerging whole-genome resequencing data in rice and soybean to expand our characterization of low diversity regions and establish a systematic methodology for improving crop populations through targeted diversity enrichment.  In addition, the characterization of haplotypes in historic lines that serve as a base for US breeding material will allow for better design of marker sets in current breeding populations.
	
## Relevant publications

[Vaughn JN, Korani W, Stein JC, Edwards JD, Peterson DG, Simpson SA, et al. Gene disruption by structural mutations drives selection in US rice breeding over the last century. PLOS Genetics. 2021;17: e1009389](https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009389)

Vaughn, J. N., & Li, Z. (2016). Genomic signatures of North American soybean improvement Inform diversity enrichment strategies and clarify the impact of hybridization. G3: Genes, Genomes, Genetics, 6(9), 2693-2705.

## Collaborators


Zenglu Li, UGA

Anna McClung, USDA-ARS

Jeremy Edwards, USDA-ARS

[Return to home page](http://genemachine.net)
