Hundreds of genetic markers for crop traits are published every year.  Such markers are very rarely the genomic locus that actually causes the observed phenotype – the “causal variant”; they are simply in physical linkage with a causal variant on the same chromosome.  It is much rarer for researchers to extend their experiments and identify these causal variants because of the substantial effort involved.  Yet knowledge of these variants can allow plant breeders and biotechnologist to dramatically expand the utility of the identified gene by 1) freeing breeders of the requirement to use  particular donor source, which may have poor agronomic qualities in linkage with the gene (soybean cyst nematode is a classic example), 2) directing site-specific mutagenesis in elite background via CRISPR/Cas9 technology, 3) avoiding situations in which, with a common marker, a new breeding population is not segregating for the marker even though it segregates for the trait, and 4) connecting the trait with more fundamental physiological process that can be modified or managed differently.

White mold is a worst-case scenario for peanut farmers in that the full extent of the damage is not evident until harvest, after the vast majority of time and money has been invested in the crop.  Soilborne sclerotia remain underground during hot, dry weather and so are not controlled by fungicide applications.  When conditions become wet, often times when the peanut canopy is fully developed and rows overlap with one another, the pathogen infects the plants.  Control of the pathogen is difficult at this point due to fungicide spray not penetrating the dense canopy, exacerbating the problem.  Further, white mold affects the pods around the crown root, which is where the most mature pods are at harvest time, devastating potential yield.  White mold is currently the most important disease for growers in the southeast United States and has cost the most in terms of yield loss and control costs for the past two decades including 37 million in 2014 (34% of total reduction in crop value and total cost).

In this project, we exploit the simplicity of QTL-seq to dramatically increase the size of populations used in conventional genetic mapping studies.  With the appropriate methodological modifications, this increased population size will allow us to get much finer genetic resolution of genomic regions controlling white-mold resistance in peanut. We also anticipate targeting other traits in peanut, including maturity and growth form, as well as comparably complex traits in rice.
	
## Relevant publications

Takagi, H., Abe, A., Yoshida, K., Kosugi, S., Natsume, S., Mitsuoka, C., ... & Innan, H. (2013). QTL‐seq: rapid mapping of quantitative trait loci in rice by whole genome resequencing of DNA from two bulked populations. The Plant Journal, 74(1), 174-183.

## Collaborators

Joshua Clevenger, Mars Corporation and UGA

Corley Holbrook, USDA-ARS and UGA

Timothy Brenneman, UGA

Anna McClung, USDA-ARS

[Return to home page](http://genemachine.net)
