# Gene Sieve: Fully exploiting model systems for gene identification

## Project Summary

Genetic variants are differences between individuals in a species that occur at specific genomic sites.  Some of these variants have no impact on an organismâ€™s performance, although they may be useful as indirect markers of a trait.  Alternatively, causal variants or, more broadly, causal genes directly affect a trait of interest.  Identification of causal genes provides breeders with a perfect marker for selection and offers critical insights into the biology of a trait.  In addition, aided by CRISPR/Cas9 technology, such insights can be used to recreate or enhance a gene in order to improve food and fiber production. 

Causal genes are identified using controlled crosses and mapping populations.  The limited scale of such experiments often results in hundreds of candidate genes.  Further refinement of this list is generally based on prioritizing mutations by how they affect their encoded protein.  While the protein-centric approach is useful, many crop species now possess a wealth of additional genomic and genome-wide association data that can assist in prioritizing candidate genes.  In addition, with the availability of numerous high-quality plant genomes, conservation across evolutionary time should be a major consideration in this effort.  There is currently no available tool in crop genomics that integrates such data and allows a plant geneticist to upload a list of possible genetic variants and get back a list of best candidate genes based on a trait of interest.

This project will involve aggregation of genome-wide association data from seven cornerstone species: Arabidopsis thaliana, rice, maize, soybean, Medicago truncatula, sorghum, and tomato, which all have stable genome sequences, extensive phenotypic data, and are broadly distributed across flowering plants.  Using a novel approach to phenotypic deposition, we plan to be able to compare QTL across species in order to make candidate-gene identification much more efficient.
	
## Collaborators

Maria Monteros, Noble Research Institute

[Return to home page](http://genemachine.net)
